### PR TITLE
Luke/circles 267 investigate docker

### PR DESCRIPTION
**Pull request by banaboi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,3 +85,6 @@ If you are doing superficial work with the frontend and do not need to communica
...
 
 To stop a docker containers that is running, run `docker-compose stop <containerName>`. This will not remove the container.
+
+### Running Circles without Docker
+It is now possible to run Circles without docker, and have your changes to the backend codebase be reflected in the development server. To do this, ensure nodemon is installed on your linux distribution by running `npm i nodemon`. Nodemon is a node package which automatically restarts an app when it detects code changes. Firstly, ensure the mongodb container is up by running `docker compose run init-mongo`. Then, to run the backend, run `MONGODB_SERVICE_HOSTNAME=localhost MONGODB_PASSWORD=<password> MONGODB_USERNAME=<username> nodemon --exec python3 runserver.py`, where `password` and `username` are your mongodb environment variables set in /env. Ensure your python3 version is set to 3.10. To run the frontend, navigate to the frontend directory and run `npm start`. All parts of the app should now be running and talking to eachother.
```
Comment (lhjt) [inline README.md]:
```suggestion
It is now possible to run Circles without docker, and have your changes to the backend codebase be reflected in the development server. To do this, ensure nodemon is installed on your linux distribution by running `npm i -g nodemon`. Nodemon is a node package which automatically restarts an app when it detects code changes. Firstly, ensure the mongodb container is up by running `docker compose run --rm init-mongo`. Then, to run the backend, run `MONGODB_SERVICE_HOSTNAME=localhost MONGODB_PASSWORD=<password> MONGODB_USERNAME=<username> nodemon --exec python3 runserver.py`, where `password` and `username` are your mongodb environment variables set in `/env`. Ensure your python3 version is set to 3.10. To run the frontend, navigate to the frontend directory and run `npm start`. All parts of the app should now be running and talking to eachother. 
```

**Review by lhjt — CHANGES_REQUESTED** [review]
Just a few changes to documentation; it looks fine otherwise. Is there a need for the `--openssl-legacy-provider` flag on the frontend though (I'm not sure what it does)

**Comment by banaboi** [comment]
The frontend was complaining without the --openssl-legacy-provider for some reason. Maybe it's an issue with my node modules install

**Review by lhjt — APPROVED** [review]

File: frontend/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "craco start",
+    "start": "craco --openssl-legacy-provider start",
```
Comment (Peedee2002) [inline frontend/package.json]: Dont do this. Instead, make sure youre using npm 16.

**Review by Peedee2002 — APPROVED** [review]
nice. One small change and it looks good to me :)

File: frontend/package.json (hunk 2 above)
Comment (banaboi) [reply frontend/package.json]: Fixed

**Pull request merged**